src/types/multiModel.js

```diff
@@ -79,6 +68,18 @@ class MultiModel extends BaseType {
         return this;
     }
 
+    map (): Array<Object> {
+        if (arguments.length === 1) {
+            return this.all().map(arguments[0]);
+        }
+
+        return this.query(arguments[0]).map(arguments[1]);
```

Comment:
is `a.get('bees').query({page: 1}).map((m) => {...})` worse than `.get('bees'.map({page: 1}, (m) => {...})`?